src/dune_rules/context.ml

```diff
@@ -223,7 +223,7 @@ end = struct
       in
       let+ s =
         Memo.of_reproducible_fiber
-          (Process.run_capture ~display:!Clflags.display ~env Strict opam args)
+          (Process.run_capture ~display:Quiet ~env Strict opam args)
```

Comment:
Here we get some opam configuration. This is internal and does not need to be reported to the user.